src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -126,6 +166,8 @@ const FilterRow: React.FC<FilterRowProps> = ({
   const handleInputChange = (value) => {
     if (value.target) {
       onInputChange(value.target.value, index);
+    } else if (value.value) {
```

Comment:
I would rather hasPath/hasProp here since if `value.value` will be `0` or empty string it's still false, same goes for `value.target`

Reply:
Yep, good point with 0 and empty string. Changed